### PR TITLE
fix: integration-tests/modules

### PR DESCRIPTION
**Pull request by adrien2p**

### CODE REVIEW

File: integration-tests/modules/package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "license": "MIT",
   "private": true,
   "scripts": {
-    "test:integration": "node --expose-gc ./../../node_modules/.bin/jest --ci --silent=false -i --detectOpenHandles --logHeapUsage --forceExit",
+    "test:integration": "node --expose-gc ./../../node_modules/.bin/jest --ci --silent=true --concurrency=50% --detectOpenHandles --logHeapUsage --forceExit -- __tests__/workflow-engine/api.ts",
```
Comment (riqwan) [inline integration-tests/modules/package.json]:
```suggestion
    "test:integration": "node --expose-gc ./../../node_modules/.bin/jest --ci --silent=true --concurrency=50% --detectOpenHandles --logHeapUsage --forceExit",
```

**Comment by olivermrbl** [comment]
@adrien2p, looks like one suite is still failing 🤔

File: integration-tests/helpers/admin-seeder.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 const Scrypt = require("scrypt-kdf")
-const { User } = require("@medusajs/medusa")
+const { User } = require("@medusajs/medusa/dist/models/user")
```
Comment (sradevski) [inline integration-tests/helpers/admin-seeder.js:2]: Why was this change needed?

**Review by sradevski — APPROVED** [review]

**Comment by adrien2p** [comment]
@olivermrbl I was looking at it but from some reason the bot still merge it :/

File: integration-tests/helpers/admin-seeder.js (hunk 2 above)
Comment (adrien2p) [reply integration-tests/helpers/admin-seeder.js:2]: Because importing from the entry file is causing issues since all import will get evaluated, meaning all models to and therefore some decorators relying on the feature flags gets applied before they are loaded

Comment (adrien2p) [reply integration-tests/helpers/admin-seeder.js:2]: this is some trouble I ve seen multiple times already and they are annoying, same when a file import from an index file, the other import gets evaluated but not necessarely at the right moment, we can re discuss that later
